drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/PhantomJSDriverFactory.java

```diff
@@ -35,17 +33,14 @@
  * PhantomJSDriver.
  *
  * @author <a href="kpiwko@redhat.com">Karel Piwko</a>
- *
+ * @author <a href="mjobanek@redhat.com">Matous Jobanek</a>
```

Comment:
No problem with it. Yeah, the question is if it brings any value; as all the changes is trackable via git, then I don't see benefit.